integrations/extensions/starter-kits/language-model-openai/README.md

```diff
@@ -0,0 +1,74 @@
+
+## Prerequisites
+
+### Create an API key
+
+1. Create an account with [OpenAI](https://openai.com/).
+1. Manage your account to [create a new API Key](https://platform.openai.com/account/api-keys). Save this API key somewhere safe and accessible. You can't view it again in your OpenAI account. If you lose this secret key, you need to generate a new one.
+
+### Create an assistant
+
+This starter kit requires that you use the [new Watson Assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-welcome-new-assistant).
+
+Create a new, empty assistant that you can use to test this starter kit. For more information, see [Adding more assistants](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-assistant-add).
+
+## Connect your assistant to OpenAI
+
+You connect your assistant by using an OpenAPI specification to add a custom extension.
+
+### Download the OpenAPI specification
+
+Download the OpenAPI specification file from the extension starter kit: [openAI.openapi.json](./openAI.openapi.json).
+
+You use this specification file to create and add the extension to your assistant.
+
+### Build and add extension
+
+1.  In your assistant, on the **Integrations** page, click **Build custom extension** and use the OpenAPI specification file to build a custom extension named `OpenAI`. For general instructions on building any custom extension, see [Building the custom extension](/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+
+1. After you build the OpenAI extension and it appears on your **Integrations** page, click **Add** to add it to your assistant. For general instructions on adding any custom extension, see [Adding an extension to your assistant](/docs/watson-assistant?topic=watson-assistant-add-custom-extension).
+
+1. In **Authentication**, choose **Bearer auth**. Copy and paste your OpenAI API key into the **Token** field.
+
+For more information about the extension parameters, see the OpenAI API Reference:
+- [Create completion](https://platform.openai.com/docs/api-reference/completions)
+- [Create chat completion](https://platform.openai.com/docs/api-reference/chat)
+
+If you need any capabilities that are in the documentation but not in the OpenAPI specification that we provided, feel free to add them to the OpenAPI specification.  The specification in the kit is intended to be a simple example of how to get started, not a comprehensive encoding of everything that API can do.
+
+## Upload sample actions
+
```

Comment:
I am adding this plus "You may also need to refresh the action Preview chat after uploading in order to get all the session variables initialized before these actions will work correctly." to address the glitch you found.  I am not sure that it will be easy for average readers to understand this, but I think it is better to put _something_ in there.